static/js/cards.js

```diff
@@ -0,0 +1,36 @@
+const getLinksCard = (cardLinks = []) => {
+	return cardLinks.map(({ title, href }) => {
+		const link = `<div class="card__text"><a href= ${href} >${title}</a></div>
```

Comment:
const link = `<div class="card__text">
  <a href="${href}">${title}</a>
</div>

Reply:
What does it mean? I didn't catch!